riscv/insns/vfunary1_vv.h

```diff
@@ -0,0 +1,18 @@
+// VFUNARY1 encoding space
```

Comment:
This should be expanded out into 2 files, one per instruction.  I've made the corresponding change to riscv-opcodes: https://github.com/riscv/riscv-opcodes/commit/0c06e7381689977480fd9ec6a1711f8eb48c6382#diff-b8de9572fc6a9a303c3300f15cf8199c

Reply:
@chihminchao

Reply:
I will follow change in riscv-opcodes.